openverse_catalog/dags/util/loader/paths.py

```diff
@@ -148,3 +159,22 @@ def _extract_media_type(tsv_file_name: Optional[str]) -> str:
     except (AttributeError, IndexError):
         media_type = "image"
     return media_type
+
+
+def _get_tsv_version(tsv_file_name: str) -> str:
+    """TSV file version can be deducted from the filename
+    v0: without _vN_ in the filename
+    v1+: has a _vN in the filename
+
+    >>>_get_tsv_version('/behance_image_20210906130355.tsv')
+    '000'
+    >>> _get_tsv_version('/jamendo_audio_v005_20210906130355.tsv')
+    '005'
+    """
+
+    version_pattern = re.compile(r"_v(\d+)_")
+    matches = re.search(version_pattern, tsv_file_name)
+    if matches:
+        return matches.group(1)
+    else:
+        return "000"
```

Comment:
We could use the walrus operator here!
```suggestion
    if match := re.search(version_pattern, tsv_file_name):
        return match.group(1)
    return "000"
```